src/file-uploader/file-uploader.module.ts

```diff
@@ -5,7 +5,7 @@ import { CheckmarkFilled16Module } from "@carbon/icons-angular/lib/checkmark--fi
 import { WarningFilled16Module } from "@carbon/icons-angular/lib/warning--filled/16";
 
 import { FileUploader } from "./file-uploader.component";
-import { File } from "./file.component";
+import { FileComponent } from "./file.component";
```

Comment:
the `NgModule` declarations need to be updated to declare `FileComponent` (and it should be added to the `exports` not sure why it's not there) ... `File` should also be exported from the module